welcome/index.adoc

```diff
@@ -46,6 +46,21 @@ a| * Overview of OpenShift architecture
 * Developing on OpenShift
 * Managing your cluster 
 |===
+
+== Available Regions
+
+{product-title} is currently available in the following regions:
+
+* East US (Public / United States)
+* East US 2 (Public / United States)
+* South Central US (Public / United States)
+* North Europe (Public / Europe)
+* West Europe (Public / Europe)
+* Southeast Asia (Public / Asia Pacific)
+* Australia East (Public / Australia)
+* UK South (Public / United Kingdom)
+* West US 2 (Public / United States)
```

Comment:
Not sure if this is copied from elsewhere in docs, but I prefer not to use spaces between forward slashes. There's not specific guidance in ISG, but they don't use them when writing things like `I/O` and `read/write`, so I personally wouldn't use them here. My 2¢.

LGTM otherwise!

Reply:
Got it. I'll update that. And yes, copied right out of an email -- good eyes!